catalog/task/collect-extra-data/0.1/collect-extra-data.yaml

```diff
@@ -0,0 +1,89 @@
+---
+apiVersion: tekton.dev/v1beta1
+kind: Task
+metadata:
+  name: collect-extra-data
+  labels:
+    app.kubernetes.io/version: "0.1"
+  annotations:
+    tekton.dev/pipelines.minVersion: "0.12.1"
+    tekton.dev/tags: release
+spec:
+  description: >-
+    Tekton task to collect extra data from release resources
+  params:
+    - name: release
+      type: string
+      description: The namespaced name of the Release
+    - name: releasePlan
+      type: string
+      description: The namespaced name of the ReleasePlan
+    - name: releasePlanAdmission
+      type: string
+      description: The namespaced name of the ReleasePlanAdmission
+  results:
+    - name: extraData
+      description: |
+        Result of merging the `extraData` field of all the release resources
+  steps:
+    - name: collect-extra-data
+      image: quay.io/hacbs-release/release-utils
+      env:
+        - name: "RELEASE"
+          value: '$(params.release)'
+        - name: "RELEASE_PLAN"
+          value: '$(params.releasePlan)'
+        - name: "RELEASE_PLAN_ADMISSION"
+          value: '$(params.releasePlanAdmission)'
+      script: |
+        #!/usr/bin/env sh
+        
+        get_kubectl_output() {
```

Comment:
I think it would be nice to pull these functions out and have them be things in the utils repo